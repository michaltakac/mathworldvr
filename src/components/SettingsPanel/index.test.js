import React from 'react'
import renderer from 'react-test-renderer'
import SettingsPanel from '.'

jest.mock('react-dom')

describe('SettingsPanel', () => {
  it('renders <a-entity>', () => {
    const tree = renderer.create(<SettingsPanel />).toJSON()
    expect(tree.type).toBe('a-entity')
  })

  it('renders the tree consistently', () => {
    const tree = renderer.create(<SettingsPanel />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
