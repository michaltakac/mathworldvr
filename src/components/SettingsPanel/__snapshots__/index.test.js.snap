// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SettingsPanel renders the tree consistently 1`] = `
<a-entity>
  <span />
  <span />
  <span />
  <span />
  <span />
  <span />
  <span />
  <span />
</a-entity>
`;
